docs/quickstarts/ansible-create-first-playbook/metadata.yml

```diff
@@ -3,6 +3,8 @@ name: ansible-create-first-playbook
 tags:
 - kind: bundle
   value: ansible
+- kind: content
+  value: documentation 
 - kind: content
   value: otherResource
```

Comment:
Hi @jself-sudoku !
All looks great to me, except we need to delete the following two lines for the content type 'otherResource', since only one content tag is permitted:
- kind: content
  value: otherResource